dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -272,12 +270,28 @@ export const Card = ({
 								) : undefined
 							}
 							commentCount={
-								showCommentCount && longCount && shortCount ? (
-									<CardCommentCount
-										containerPalette={containerPalette}
-										format={format}
-										long={longCount}
-										short={shortCount}
+								discussionId ? (
+									<Link
+										// This a tag is initially rendered empty. It gets populated later
+										// after a fetch call is made to get all the counts for each Card
+										// on the page with a discussion (see FetchCommentCounts.tsx)
+										data-name="comment-count-marker"
+										data-discussionid={discussionId}
```

Comment:
If you use `data-discussion-id`, it will [be available as `discussionId` on the `dataset`, with proper `camelCase`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#name_conversion).

```suggestion
										data-discussion-id={discussionId}
```

Reply:
Thanks, great suggestion - I'll do this now